### ISSUE TITLE
Tkinter spinbox test failures on Tk 8.7

### ISSUE THREAD

**Issue by chrstphrchvz** [python/cpython#104856]
Tk 8.7 spinboxes allow setting `to` to an amount less than `from`, and setting `from` to an amount greater than `to`, by swapping the values rather than raising an exception; see https://core.tcl-lang.org/tk/info/841280 . This causes two tests to fail:

```
======================================================================
FAIL: test_configure_from (test.test_tkinter.test_widgets.SpinboxTest.test_configure_from)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/user/git/cpython/Lib/test/test_tkinter/test_widgets.py", line 490, in test_configure_from
    self.checkInvalidParam(widget, 'from', 200,
  File "/Users/user/git/cpython/Lib/test/test_tkinter/widget_tests.py", line 60, in checkInvalidParam
    with self.assertRaises(tkinter.TclError) as cm:
AssertionError: TclError not raised

======================================================================
FAIL: test_configure_to (test.test_tkinter.test_widgets.SpinboxTest.test_configure_to)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/user/git/cpython/Lib/test/test_tkinter/test_widgets.py", line 501, in test_configure_to
    self.checkInvalidParam(widget, 'to', -200,
  File "/Users/user/git/cpython/Lib/test/test_tkinter/widget_tests.py", line 60, in checkInvalidParam
    with self.assertRaises(tkinter.TclError) as cm:
AssertionError: TclError not raised
```

**Comment by terryjreedy** [comment]
These assertion failures can be fixed by the same mechanism used to fix assertion failures in #104855, and on the same PR.

**Comment by serhiy-storchaka** [comment]
Fixed by #120824.
